src/main/java/org/launchcode/liftoffproject/models/Parent.java

```diff
@@ -65,7 +65,7 @@ public void setChildren(List<Child> children) {
         this.children = children;
     }
 
-    public List<Chore> getChores() {
+    public List<Chore> getChores(String username) {
```

Comment:
This might be the right strategy, but we should use the `id` for the Child (since the `username` is one step removed and actually belongs to the ChildUser object instead of the Child)